### PR TITLE
INDY-2068: TAA AML auth rules

### PR DESCRIPTION
**Pull request by ArtObr**
Signed-off-by: ArtObr <artemobruchnikov@gmail.com>

### CODE REVIEW

File: indy_node/test/auth_rule/test_auth_map.py [hunk 1]
```diff
@@ -38,6 +38,14 @@ def test_auth_map_txn_author_agreement():
         assert rule_str in auth_map.auth_map.keys()
```
Comment (ashcherbakov) [inline indy_node/test/auth_rule/test_auth_map.py]: Please add the following tests as well:

   - a unit test to indy_common/test/auth (test_txn_author_agreement_aml_with_new_auth_map.py)
   - an integration test for auth rule changing into indy_node/test/auth_rule/auth_framework

Comment (ArtObr) [reply indy_node/test/auth_rule/test_auth_map.py]: ok

File: design/txn_author_agreement.md [hunk 2]
```diff
@@ -91,6 +92,7 @@ Add new version of AML.
     "reqId": INT,
     "operation": {
         "type": INT,
+        "version": <str>
```
Comment (ashcherbakov) [inline design/txn_author_agreement.md]: Please mention getting AML by version `GET_TXN_AUTHOR_AGREEMENT_AML` section.

Comment (ashcherbakov) [inline design/txn_author_agreement.md]: Please describe how AML is stored in state

Comment (ArtObr) [reply design/txn_author_agreement.md]: ok

Comment (ArtObr) [reply design/txn_author_agreement.md]: ok

**Comment by ArtObr** [comment]
test this please

**Pull request merged**